### PR TITLE
[KH2] Add new Poptracker Pack to the KH2 Setup Guide

### PR DESCRIPTION
**Pull request by palex00**
## What is this fixing or adding?
Adding the new Poptracker Pack developed by @Ugh-Sunlight and me. Both have full access to the repository and can continue the work if the other dies.

The pack is based on the logic sheet and is still being tested thoroughly but it already extremly accurate. The maintainer of KH2 is aware of the pack and it is pinned in the channel.

The pack is currently not yet fully done (1.0.0) because some Icons are still placeholder but before 0.5.1 drops that will be done.

## How was this tested?
Visual look at the Setup Guide. Pack itself is being tested [here](https://discord.com/channels/731205301247803413/1295193435418005554)

## If this makes graphical changes, please attach screenshots.
image
image-1

### CODE REVIEW

**Comment by palex00** [comment]
@JaredWeakStrike

File: worlds/kh2/docs/setup_en.md
Comment (Exempt-Medic) [inline worlds/kh2/docs/setup_en.md:1]: The capitalization of PopTracker is inconsistent

File: worlds/kh2/docs/setup_en.md [hunk 1]
```diff
@@ -73,10 +79,24 @@ Enter `The room's port number` into the top box <b> where the x's are</b> and pr
 - Run the game in windows/borderless windowed mode. Fullscreen is stable but the game can crash if you alt-tab out.
 - Make sure to save in a different save slot when playing in an async or disconnecting from the server to play a different seed
 
-<h2 style="text-transform:none";>Logic Sheet</h2>
+<h2 style="text-transform:none";>Poptracker Autotracking & Logic Sheet</h2>
```
Comment (Exempt-Medic) [inline worlds/kh2/docs/setup_en.md:82]: This should probably be reversed to match the order in the section
```suggestion
<h2 style="text-transform:none";>Logic Sheet & Poptracker Autotracking</h2>
```

File: worlds/kh2/docs/setup_en.md
Comment (palex00) [reply worlds/kh2/docs/setup_en.md:1]: PR'd

**Review by nicholassaylor — APPROVED** [review]
Read through and everything looks good now

**Review by Exempt-Medic — APPROVED** [review]
LGTM, displays and reads well and matches the formatting in the rest of the file

**Comment by JaredWeakStrike** [comment]
lgtm

**Review by NewSoupVi — APPROVED** [review]

**Pull request merged**